sdkproject/Assets/Mapbox/Unity/Editor/GeocodeAttributeSearchWindow.cs

```diff
@@ -125,9 +125,12 @@ void HandleUserInput(string searchString)
 
 		void HandleGeocoderResponse(ForwardGeocodeResponse res)
 		{
-			_features = res.Features;
+			if (res != null)
+			{
+				_features = res.Features;
```

Comment:
Should you clear `_features` when submitting a new search?

Reply:
Right, good catch! I will test and correct my changes, thank you

Reply:
For some reason, I'm not able to test any cases where the geocoding works (probably because of my network), but I called the `Clear` function on `_features` to make sure it's empty when nothing is found, would that make any sense?

Reply:
Ah, so it was actually being reinitialized here: https://github.com/thedrummeraki/mapbox-unity-sdk/blob/07ca43b1b5ca0ccf6fa0ba14f12aee5b02dd623f/sdkproject/Assets/Mapbox/Unity/Editor/GeocodeAttributeSearchWindow.cs#L116. I think you can remove the `else` branch you just added. My bad.

Reply:
Ok cool, didn't realize this myself, I just removed the `else` clause, thanks